lib/l10n/app_fr.arb

```
@@ -1886,5 +1886,17 @@
   "unlinkTaskTitle": "Délier la tâche",
   "viewMenuTitle": "Affichage",
   "whatsNewDoneButton": "Terminé",
-  "whatsNewSkipButton": "Ignorer"
-}
\ No newline at end of file
+  "whatsNewSkipButton": "Ignorer",
+  "agentEvolutionVersionProposed": "Version proposée",
```

RuboCop (1.84.2)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)